pkg/virt-api/rest/expand.go

```diff
@@ -49,6 +49,20 @@ func (app *SubresourceAPIApp) ExpandSpecRequestHandler(request *restful.Request,
 		return
 	}
 
+	requestNamespace := request.PathParameter("namespace")
+	if requestNamespace == "" {
+		writeError(errors.NewBadRequest("The request namespace must not be empty"), response)
```

Comment:
Not critical but I think you missed a test here, can you address in a follow up?

Reply:
You mean a test for an empty `request.PathParameter("namespace")`?